src/main/java/org/spongepowered/common/SpongeCatalogType.java

```diff
@@ -78,7 +78,7 @@ protected ToStringHelper toStringHelper() {
                 .add("name", getName());
     }
 
-    public static abstract class Translatable extends SpongeCatalogType implements org.spongepowered.api.text.translation.Translatable {
+    public abstract static class Translatable extends SpongeCatalogType implements org.spongepowered.api.text.translation.Translatable {
```

Comment:
Okay. I will undo these then and I think remove the JLS order check style altogether as modifier order does not matter at all and it is hard to guarantee people ordering in the same way anyway. Will just get messed up again.